netbox/dcim/tables/devices.py

```diff
@@ -933,6 +933,9 @@ class InventoryItemTable(DeviceComponentTable):
     discovered = columns.BooleanColumn(
         verbose_name=_('Discovered'),
     )
+    parent = tables.LinkColumn(
```

Comment:
FYI LinkColumn has been [deprecated](https://django-tables2.readthedocs.io/en/latest/pages/api-reference.html#django_tables2.columns.LinkColumn): The recommendation is now to use `Column(linkify=True)` instead.